lib/install.ps1

```diff
@@ -820,7 +842,7 @@ function env_set($manifest, $dir, $global) {
     if($manifest.env_set) {
         $manifest.env_set | gm -member noteproperty | % {
             $name = $_.name;
-            $val = format $manifest.env_set.$($_.name) @{ "dir" = $dir }
+            $val = format $manifest.env_set.$($_.name) @{ "dir" = $dir; "persist_dir" = $persist_dir }
```

Comment:
If prepare_env_path can be removed, it would be good to not have to change this either.